packages/legend-studio/src/stores/graphModifier/DSLService_GraphModifierHelper.ts

```diff
@@ -41,7 +45,12 @@ export const service_initNewService = action(
   },
 );
 export const service_setExecution = action(
-  (service: Service, value: ServiceExecution): void => {
+  (
+    service: Service,
```

Comment:
I think you missed a bunch of methods in this file, such as `pureSingleExecution_setRuntime`